doc/man7/EVP_PKEY-ML-DSA.pod

```diff
@@ -1,5 +1,7 @@
 =pod
 
+=encoding utf-8
```

Comment:
Why is this necessary? (it is not in any other pod file)

Reply:
Actually it is.

Reply:
My concern was if this is something new then it might not output correctly  of the web pages.